example/package.json

```diff
@@ -20,9 +20,11 @@
     "@storybook/addon-essentials": "^6.2.9",
     "@storybook/addon-links": "^6.2.9",
     "@storybook/vue": "^6.2.9",
+    "@vue/cli-plugin-babel": "~4.5.0",
+    "@vue/cli-plugin-typescript": "~4.5.0",
     "@vue/cli-service": "~4.5.0",
-    "@storybook/testing-vue": "../",
```

Comment:
Had to import directly from the `dist` folder because this was causing yarn to re-install over and over the dev dependencies in a loop. Not sure why 🤔 .